library/core/src/alloc/global.rs

```diff
@@ -157,14 +157,14 @@ pub unsafe trait GlobalAlloc {
     ///
     /// # Safety
     ///
-    /// This function is unsafe because undefined behavior can result
-    /// if the caller does not ensure all of the following:
+    /// The caller has to ensure that
```

Comment:
```suggestion
    /// The caller must ensure that:
```

`must` is a bit stronger than `has to`